enterprise/cmd/frontend/internal/authz/resolvers/resolver_test.go

```diff
@@ -2579,3 +2579,78 @@ func mustParseTime(v string) time.Time {
 func intPtr(v int) *int              { return &v }
 func timePtr(v time.Time) *time.Time { return &v }
 func stringPtr(v string) *string     { return &v }
+
+func TestResolver_PermissionsSyncingStats(t *testing.T) {
+	t.Run("authenticated as non-admin", func(t *testing.T) {
+		user := &types.User{ID: 42}
+
+		users := database.NewStrictMockUserStore()
+		users.GetByCurrentAuthUserFunc.SetDefaultReturn(user, nil)
+		users.GetByIDFunc.SetDefaultReturn(user, nil)
+
+		db := edb.NewStrictMockEnterpriseDB()
+		db.UsersFunc.SetDefaultReturn(users)
+
+		ctx := actor.WithActor(context.Background(), &actor.Actor{UID: user.ID})
+		_, err := (&Resolver{db: db}).PermissionsSyncingStats(ctx)
+		if want := auth.ErrMustBeSiteAdmin; err != want {
+			t.Errorf("err: want %q but got %v", want, err)
+		}
+	})
+
+	t.Run("authenticated as site-admin", func(t *testing.T) {
```

Comment:
We usually name positive cases (i.e. not when site-admin check fails) differently
```suggestion
	t.Run("successfully query all permissionsSyncingStats", func(t *testing.T) {
```

Reply:
fixed